src/main/resources/admin/tools/content-type-manager/components/structure/component-item-set.html

```diff
@@ -77,8 +77,9 @@
           return `${this.path}elements[${index}]`;
         },
 
-        removeSet() {
-          this.$store.commit("removeSchemaInJson", { path: this.path });
+        async removeSet() {
```

Comment:
This is repeated quite a lot, can it be refactored into a single function that is used in multiple places?